backend/clubs/serializers.py

```diff
@@ -849,6 +850,24 @@ class Meta:
         fields = ("club", "person")
 
 
+class FavoriteTimeSerializer(serializers.ModelSerializer):
+    """
+    Used by clubs to get a list of people who have favorited their club as
+    well as when those people have favorited (for analytics).
```

Comment:
We don't actually want to tell clubs who has marked their club as favorite, favorites should be for student purposes only.

If students want to give that information to the club, they can subscribe to the club.